js/app.js

```diff
@@ -0,0 +1,55 @@
+"use strict";
+
+(function(){
+  angular
+  .module("wdiRadio", [
+    "ui.router",
+    "ng-token-auth",
+    "songs"
+  ])
+  .config([
+    "$authProvider",
+    AuthConfigFunction
+  ])
+  .config([
+    "$stateProvider",
+    RouterFunction
+  ]);
+  
+  function AuthConfigFunction($authProvider){
+    $authProvider.configure({
+      apiUrl: "http://localhost:3000"
+    });
+  }
+  function RouterFunction($stateProvider){
+    $stateProvider
+      .state("radioIndex",{
+        url:"/welcome",
+        templateUrl:"js/welcome.html"
+      })
+      .state("signin", {
+        url: "/signin",
+        templateUrl: "js/users/signin.html",
+        controller: "SessionsController",
+        controllerAs: "SessionsViewModel"
```

Comment:
So, you did not need to implement user authentication in this lab assignment- glad if you got it working though!